pytorch_lightning/callbacks/stochastic_weight_avg.py

```diff
@@ -102,12 +102,10 @@ def __init__(
         if isinstance(swa_epoch_start, float) and not (0 <= swa_epoch_start <= 1):
             raise MisconfigurationException(err_msg)
 
-        if (
-            swa_lrs is not None and (
-                not isinstance(swa_lrs, (float, list)) or isinstance(swa_lrs, float) and swa_lrs <= 0
-                or isinstance(swa_lrs, list) and not all(lr > 0 and isinstance(lr, float) for lr in swa_lrs)
-            )
-        ):
+        _wrong_type = not isinstance(swa_lrs, (float, list))
+        _wrong_float = isinstance(swa_lrs, float) and swa_lrs <= 0
+        _wrong_list = isinstance(swa_lrs, list) and not all(lr > 0 and isinstance(lr, float) for lr in swa_lrs)
+        if (swa_lrs is not None and (_wrong_type or _wrong_float or _wrong_list)):
```

Comment:
No need for underscore

```suggestion
        wrong_type = not isinstance(swa_lrs, (float, list))
        wrong_float = isinstance(swa_lrs, float) and swa_lrs <= 0
        wrong_list = isinstance(swa_lrs, list) and not all(lr > 0 and isinstance(lr, float) for lr in swa_lrs)
        if (swa_lrs is not None and (wrong_type or wrong_float or wrong_list)):
```